test/tests/functional/pbs_mom_hook_sync.py

```diff
@@ -175,10 +175,10 @@ def test_momhook_to_serverhook_with_resume(self):
             'to %s.*' % self.momB.hostname,
             starttime=now, max_attempts=10, regexp=True)
 
-        # the higher the number, the earlier the line appears in the log
-        self.assertTrue(match_delete[0] > match_sent1[0])
-        self.assertTrue(match_delete[0] > match_sent2[0])
-        self.assertTrue(match_delete[0] > match_sent3[0])
+        # the lower the number, the earlier the line appears in the log
```

Comment:
while we are changing the comments, I suggest we remove the extra "the"s in the comment -
# lower the number, earlier the line appears in the log